File: generator/src/test/java/ch/bailu/gtk/TestPrimitivesTable.java
package ch.bailu.gtk;

import org.junit.jupiter.api.Test;

import ch.bailu.gtk.table.PrimitivesTable;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class TestPrimitivesTable {
    @Test
    void test() {
        assertEquals(true, PrimitivesTable.INSTANCE.contains("gdouble"));
        assertEquals("double", PrimitivesTable.INSTANCE.convert("gdouble"));


        assertEquals(true, PrimitivesTable.INSTANCE.contains("double"));
        assertEquals("double", PrimitivesTable.INSTANCE.convert("double"));

    }
}

File: generator/src/test/kotlin/ch/bailu/gtk/TestPrimitivesTable.kt
package ch.bailu.gtk

import ch.bailu.gtk.table.PrimitivesTable.contains
import ch.bailu.gtk.table.PrimitivesTable.convert
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class TestPrimitivesTable {
    @Test
    fun test() {
        Assertions.assertEquals(true, contains("gdouble"))
        Assertions.assertEquals("double", convert("gdouble"))
        Assertions.assertEquals(true, contains("double"))
        Assertions.assertEquals("double", convert("double"))
    }
}
